playbook/workshops/contrib/ajar/README.md

```diff
@@ -13,10 +13,10 @@ The workshop will be divided into several parts (check indicates workshop has
 been written):
 
 - [x] [__Part 1__](#part-1) – display the user's cell on screen.
-- [ ] __Part 2__ – convert the code from part 1 to use functions.
-- [ ] __Part 4__ – make the user's cell follow the mouse cursor.
+- [x] [__Part 2__](#part-2) – convert the code from part 1 to use functions.
```

Comment:
Many keyboards don't provide an easy way to enter a dash character. Would you please replace these en-dashes to with hyphens? (`–` to `-`)

I'll make sure to add  that in our markdown styleguide.